sections/csgcn/csgcn-chapter.tex

```diff
@@ -309,7 +309,7 @@ \subsection{Model intuition}\label{subsec:csgcn_adj_intuition}
 This approach is unable to capture which specific contexts the different interactions occurred in.
 It only captures whether or not the user or item has had an interaction in the given context.
 This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single embedding instead of having an embedding for each context value and item pair.
-The intuition is that since context is now modeled similarly to side-information by the addition of nodes, it makes sense to include it in the convolution.
+The intuition is that since context is now modeled similarly to side-information by the addition of nodes, it makes sense to include it in the convolution, since they will be present as neighboring nodes to items and users in the pentapartite graph.
```

Comment:
```suggestion
The intuition is that since context is now modeled similarly to side-information by the addition of nodes, it makes sense to include it in the convolution, since they will be present as neighboring nodes to items and users in the graph.
```